core-bundle/src/Resources/contao/modules/ModuleSearch.php

```diff
@@ -313,13 +315,61 @@ protected function compile()
 					$objTemplate->hasContext = true;
 				}
 
+				$this->addImageToTemplateFromSearchResult($arrResult[$i], $objTemplate);
+
 				$this->Template->results .= $objTemplate->parse();
 			}
 
 			$this->Template->header = vsprintf($GLOBALS['TL_LANG']['MSC']['sResults'], array($from, $to, $count, $strKeywords));
 			$this->Template->duration = System::getFormattedNumber($query_endtime - $query_starttime, 3) . ' ' . $GLOBALS['TL_LANG']['MSC']['seconds'];
 		}
 	}
+
+	protected function addImageToTemplateFromSearchResult(array $result, Template $template): void
+	{
+		$template->hasImage = false;
+
+		if (!\array_key_exists('meta', $result) || null === $result['meta'])
```

Comment:
```suggestion
		if (!isset($result['meta']))
```

See https://3v4l.org/GDsMU